FrameworkSystem/Client/BundleDeliveryClient.py

```diff
@@ -49,8 +70,19 @@ def __setHash(self, bundleID, dirToSyncTo, bdHash):
       self.log.error("Could not save hash after synchronization", "%s: %s" % (fileName, str(e)))
 
   def syncDir(self, bundleID, dirToSyncTo):
+    """ Synchronize directory
+
+        :param str bundleID: bundle ID
+        :param str dirToSyncTo: path to sync directory
+
+        :return: S_OK(bool)/S_ERROR()
+    """
     dirCreated = False
-    if not os.path.isdir(dirToSyncTo):
+    if so.path.isdir(dirToSyncTo):
```

Comment:
I guess it should be

```suggestion
    if os.path.isdir(dirToSyncTo):
```

Reply:
Oh, yes .. thank you )